.travis.yml

```diff
@@ -4,23 +4,33 @@ python:
   - 3.3
   - 3.4
 sudo: false
-before_install:
+install:
   - lsb_release -a # get info on the operating system
-  - pip install --upgrade pip
-  - pip install --upgrade setuptools
+  - if [[ "${TRAVIS_PYTHON_VERSION:0:1}" == '2' ]]; then
+      wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  # Useful for debugging any issues with conda
+  - conda info -a
+  # Replace dep1 dep2 ... with your dependencies
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy==1.8.2 scipy==0.14.0 matplotlib==1.3.1 basemap==1.0.7 flake8 future lxml sqlalchemy mock nose gdal
```

Comment:
Only thing is you added `gdal` here when it's not technically a dependency just yet...